swins/layers/window_attn.py

```diff
@@ -137,7 +137,7 @@ def call(
 
         x = tf.matmul(attn, v)
         x = tf.transpose(x, perm=[0, 2, 1, 3])
-        x = tf.reshape(x, (B_, N, -1))
+        x = tf.reshape(x, (B_, N, C))
```

Comment:
Are we retrieving `C`?

Reply:
yes, see https://github.com/sayakpaul/swin-transformers-tf/blob/main/swins/layers/window_attn.py#L112